src/digest/md5.cr

```diff
@@ -105,25 +105,25 @@ class Digest::MD5 < Digest::Base
   def ff(a, b, c, d, x, s, ac)
     a += f(b, c, d) + x + ac.to_u32
     a = rotate_left a, s
-    a += b
+    a + b
```

Comment:
ditto. I don't think that doing transformations that work only because the statement is the last in body, hence the result value, is good to do.

Reply:
Sorry, didn't get it. The result value `a + b` just the same as `a += b`, but don't assign `a` once again.

Reply:
Actually this is far better ~ it returns the result idiomatically without having to work out what will be returned

Reply:
@bcardiff it shows intention, and cuts out the 'what will happen to a' ~ which is idiomatic Crystal as far as I understand.